using-mobile-extensions/adobe-edge-consent/README.md

```diff
@@ -0,0 +1,158 @@
+# Adobe Experience Platform Edge Consent
+
+The Adobe Experience Platform Consent Collection mobile extension enables consent preferences collection from your mobile app when using the Adobe Experience Platform [https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/mobile-core](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/mobile-core) and the [Edge Network extension](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge).
```

Comment:
Can you update the core link as well to show [Adobe Experience Platform Mobile SDK](https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/mobile-core)